ifrs17-template/Test/Data/IfrsVariableBenchmarks/BM_CH_2021_3__BE_Projection_DTP.csv

```diff
@@ -0,0 +1,5 @@
+@@Main
+Month,ReportingNode,Scenario,Year
+3,CH,,2021
+@@BE
+AccidentYear,AmountType,AocType,DataNode,EconomicBasis,EstimateType,Novelty,Value,Value1,Value10,Value11,Value12,Value2,Value3,Value4,Value5,Value6,Value7,Value8,Value9
```

Comment:
This is clearly wrong. And the fact that the test is green also shows that something is wrong with the test